build.sbt

```diff
@@ -34,7 +29,7 @@ lazy val files = project
 
 lazy val mqtt = project
   .in(file("mqtt"))
-  .enablePlugins()
+  .enablePlugins(AutomateHeaderPlugin)
```

Comment:
Ah, didn't remember we did the defaults using an auto plugin, thanks for clarifying.